values.yaml

```diff
@@ -1,55 +1,75 @@
 # Available parameters and their default values for the Consul chart.
 
+# global holds values that affect multiple components of the chart.
 global:
-  # enabled is the master enabled switch. Setting this to true or false
-  # will enable or disable all the components within this chart by default.
-  # Each component can be overridden using the component-specific "enabled"
-  # value.
+  # enabled is the master enabled/disabled setting.
+  # If true, servers, clients, Consul DNS and the Consul UI will be enabled.
```

Comment:
specified exactly which components will be enabled because it previously said they'd all be enabled but things like meshgateway, connect wouldn't be so thought it best to be clear

Reply:
You could also set `syncCatalog.enabled`, `connectInject.enabled` (didn't check others) to `-`, and they will respect the `global.enabled` value.

Reply:
True. I just tried to document that here
>   # connectInject and syncCatalog's enabled keys can be set to "-" to inherit from this value as well

But it didn't really fit imo. If we have those docs on those keys do you think that's sufficient?

Reply:
I'll merge this and update based on your response.

Reply:
I'm going to merge this as is but come back and update based on your response.